app/views/admin/organizations/index.html.slim

```diff
@@ -3,19 +3,48 @@
 
 = render 'layouts/admin/tables'
 
-= grid(@organizations_grid, show_filters: :when_filtered) do |g|
+= define_grid(@organizations_grid, show_filters: :when_filtered) do |g|
 
-  - g.blank_slate content_tag(:div, "No records found", class: 'alert alert-info')
+  - g.blank_slate do
+    - @empty_list = true
+    - content_tag(:div, "No records found", class: 'alert alert-info')
 
   - g.column name: 'ID', attribute: 'id'
 
-  - g.column name: 'Name', attribute: 'name'
+  - g.column name: 'Name', attribute: 'name', detach_with_id: :name_filter
 
-  - g.column name: 'City', attribute: 'city'
+  - g.column name: 'City', attribute: 'city', detach_with_id: :city_filter,
+    custom_filter: :auto
 
-  - g.column name: 'Country' do |organization|
+  - g.column name: 'Country', attribute: 'name', assoc: :country,
+    custom_filter: :auto, detach_with_id: :country_filter do |organization|
     - organization.country.try(:name)
 
   - g.column name: 'Created', attribute: 'created_at'
 
   - g.column name: 'Updated', attribute: 'updated_at'
+
+- unless @empty_list
+  header.block
+    div.row.filters
```

Comment:
I am noticing a lot of your `row` divs are not inside a `container`. See http://getbootstrap.com/css/#grid-intro. That is why, I think, you are getting the padding to the left of 'Name' here:

![image](https://cloud.githubusercontent.com/assets/702320/12625055/f8b51b12-c4ff-11e5-9857-00fd76687e9c.png)